From 0521d2a56914ab049aa8980a0b107e64e0d361ee Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Mon, 29 Sep 2014 14:24:53 -0700
Subject: [PATCH 159/202] MIPS: OCTEON: Try to allocate at least 256MB of DMA32
 memory.

The last attempt (of 64MB) was not sufficient.  We need more than
128MB, so go all the way to 256.

Signed-off-by: David Daney <david.daney@cavium.com>
[Original patch taken from Cavium SDK 3.1.1-544]
Signed-off-by: Bin Jiang <bin.jiang@windriver.com>
---
 arch/mips/cavium-octeon/setup.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/mips/cavium-octeon/setup.c b/arch/mips/cavium-octeon/setup.c
index 368b379..8026052 100644
--- a/arch/mips/cavium-octeon/setup.c
+++ b/arch/mips/cavium-octeon/setup.c
@@ -1017,8 +1017,8 @@ void __init plat_mem_setup(void)
 	limit_min = 0;
 	while ((boot_mem_map.nr_map < BOOT_MEM_MAP_MAX)
 		&& (total < MAX_MEMORY)) {
-		/* Try to get 64MB of 32=bit memory */
-		if (total >= 64 * (1<<20))
+		/* Try to get 256MB of 32-bit memory */
+		if (total >= 256 * (1<<20))
 			limit_max = ~0ull;		/* unlimitted */
 
 		memory = cvmx_bootmem_phy_alloc(mem_alloc_size,
-- 
1.8.2.1

